wordpress-mobile/WordPress-Android
# Issue/8644 remove unused activity

### Fix
Remove unused code replaced by the full-screen dialog interface for ***Quick Start*_ as described in https://github.com/wordpress-mobile/WordPress-Android/issues/8644.

#### Note
The old seven tasks excluding ***Create your site*** will start the task tutorial, but the new four tasks will not.  The old seven task are listed below:
- Create your site
- View your site
- Browse themes
- Customize your site
- Add social media account
- Publish a post
- Follow other sites

### Test
0. Select site with Quick Start in progress.
1. Go to ***Sites*** tab.
2. Notice ***Next Steps*** view is shown.
3. Tap ***Customize Your Site*** type.
4. Notice ***Customize Your Site*** tasks are shown.
5. Tap old task in list.
6. Notice task tutorial is shown.
7. Follow task tutorial.
8. Tap ***Customize Your Site*** type.
9. Notice task from Step 7 is completed.
10. Tap new task in list.
11. Notice task tutorial is not shown.
12. Tap ***Grow Your Audience*** type.
13. Notice ***Grow Your Audience*** tasks are shown.
14. Tap old task in list.
15. Notice task tutorial is shown.
16. Follow task tutorial.
17. Tap ***Grow Your Audience*** type.
18. Notice task from Step 16 is completed.
19. Tap new task in list.
20. Notice task tutorial is not shown.

### Review
Only one developer is required to review these changes, but anyone can perform the review.
